airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/ConditionSection.tsx

```diff
@@ -24,7 +25,7 @@ const GroupLabel = styled(Label)`
 `;
 
 const ConditionControls = styled.div`
-  padding-top: 25px;
+  padding-top: 20px;
 `;
```

Comment:
Since we have a styles folder, we should get rid of ConditionControls and just use a style.